api/who/who.proto

```diff
@@ -25,10 +25,23 @@ message PutLocationRequest {
   string isoCountryCode = 12;
 }
 
+message JurisdictionId {
+  JurisdictionType jurisdictionType = 2;
+  // empty for global stats, ISO 3166-1 alpha-2 (uppercase) code for country
+  string code = 1;
+}
+
+message GetCaseStatsRequest {
+  // Request any sub-global jurisdictions 
+  repeated JurisdictionId jurisdictions = 1;
+}
+
 message GetCaseStatsResponse {
   CaseStats globalStats = 1;
```

Comment:
Deprecate `globalStats` since it can be included as an instance of `jurisdictionStats`. Nice thing to do now as there's no backward compatibility to worry about. Even my TestFlight install timed out after 90 days.